tests/pl/homeassistant_HassGetState.yaml

```diff
@@ -1,126 +1,152 @@
+          - "Wentylatora stołowego"
+    response: "Stan Wentylatora stołowego to wyłączone"
 
   - sentences:
-      - "stan termostatu?"
+      - "Stan temperatury na zewnątrz?"
     intent:
       name: HassGetState
       slots:
-        name: "Termostatu"
-    response: "Stan Termostatu to 21 °C"
+        name:
+          - "Temperatura na zewnątrz"
+          - "Temperatury na zewnątrz"
+    response: "Stan Temperatury na zewnątrz to 32 °C"
 
   - sentences:
-      - "jaki jest status termostatu?"
+      - "Jaki jest stan temperatury na zewnątrz?"
     intent:
       name: HassGetState
       slots:
-        name: "Termostatu"
-    response: "Status Termostatu to 21 °C"
+        name:
+          - "Temperatura na zewnątrz"
+          - "Temperatury na zewnątrz"
+    response: "Stan Temperatury na zewnątrz to 32 °C"
 
   - sentences:
-      - "status termostatu?"
+      - "Status temperatury na zewnątrz?"
     intent:
       name: HassGetState
       slots:
-        name: "Termostatu"
-    response: "Status Termostatu to 21 °C"
+        name:
+          - "Temperatura na zewnątrz"
+          - "Temperatury na zewnątrz"
+    response: "Status Temperatury na zewnątrz to 32 °C"
 
   - sentences:
-      - "czy lampa główna jest włączona?"
+      - "Czy lampa główna jest włączona?"
+      - "Czy jest włączona lampa główna?"
     intent:
       name: HassGetState
       slots:
         name: "Lampa główna"
         state: "on"
-    response: "Nie, lampa główna jest off"
+    response: "Nie, Lampa główna jest wyłączona"
 
   - sentences:
-      - czy są jakieś przełączniki włączone w kuchni?
-      - czy w kuchni są jakieś włączone przełączniki?
+      - "Czy są jakieś przełączniki włączone w kuchni?"
+      - "Czy są włączone jakieś kontakty w kuchni?"
+      - "Czy w kuchni są jakieś włączone przełączniki?"
+      - "Czy w kuchni są włączone jakieś kontakty?"
     intent:
       name: HassGetState
       slots:
         area: "Kuchni"
-        domain: "switch"
+        domain: switch
         state: "on"
     response: "Nie"
 
   - sentences:
-      - czy jest wentylator włączony w kuchni?
-      - czy w kuchni jest jakiś włączony wentylator?
+      - "Czy w kuchni jest włączony wentylator?"
+      - "Czy jest wentylator włączony w kuchni?"
+      - "Czy jest włączony wentylator w kuchni?"
+      - "Czy w kuchni jest jakiś włączony wentylator?"
     intent:
       name: HassGetState
       slots:
         area: "Kuchni"
-        domain: "fan"
+        domain: fan
         state: "on"
     response: "Nie"
 
   - sentences:
-      - czy wszystkie kontakty są włączone?
+      - "Czy wszystkie kontakty są włączone?"
+      - "Czy są włączone wszystkie kontakty?"
     intent:
       name: HassGetState
       slots:
-        domain: "switch"
+        domain: switch
         state: "on"
-    response: "Nie, Ekspres do kawy i Przełącznik kuchenny nie są"
+    response: "Nie, Ekspresu do kawy i Przełącznik kuchenny nie są"
```

Comment:
This is correct test, but because I need use two name of unit (Ekspres and Ekspresu) https://github.com/witold-gren/intents/blob/main/tests/pl/_fixtures.yaml#L186 the test will choose full version of switch name https://github.com/witold-gren/intents/blob/main/tests/pl/homeassistant_HassGetState.yaml#L110. You can find test for this sentence in this place https://github.com/witold-gren/intents/blob/main/tests/pl/homeassistant_HassGetState.yaml#L22 If you have any idea how to separate it, let me know... I'll fix it..

Reply:
Ohhh, now I see the problem here, no worries, we can iterate over this later.